docs/list.md

```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Sign in with [Github here](https://devict-hacktoberfest.herokuapp.com/) so they can track your PRs.
 - **Notes**: You need to create at least two PRs in any of the  Wichita developer community organizations listed [here](https://devict-hacktoberfest.herokuapp.com/)
 
+#### **devtron**
+
+- **Swag**: 1st Rank : Devtron's Limited Edition Swag Kit
+          2nd Rank : Devtron's Limited Edition Diary + stickers
+          3rd Rank : Devtron's Limited Edition stickers
+- **Requirements**:  Star the [Devtron](https://github.com/devtron-labs/devtron) repo on Github
+                     Follow us on [Twitter](https://twitter.com/DevtronL)
+                     Make some art and share it on Twitter. Tag @DevtronL and use #Hacktoberfest, #Devtron on your post.
+- **How to sign up**: Open this [Github Repo](https://github.com/devtron-labs/devtron) ,Fork it and star it . When submitting a PR please add Hacktoberfest in the PR Description. This would give us a confirmation on whether the PR is for Hacktoberfest.
+- **Notes**: Read the contributions guidelines caredfully and for such opportunites in devtron [Click Here](https://github.com/devtron-labs/hacktoberfest2022/blob/main/README.md#book-contribution-guidelines)
+
```

Comment:
Thank you for adding Devtron to the Swag List! I'd encoruage you to look at the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing/) to make sure you're formatting things correctly (as well as other entries on the List). 

Netlify will also provide you with a build preview with each new commit, so feel free to use those previews to make sure that things look the way you want them to.